halogen/schema.py

```diff
@@ -120,7 +128,7 @@ class Attr(object):
 
     creation_counter = 0
 
-    def __init__(self, attr_type=None, attr=None, required=True, **kwargs):
+    def __init__(self, attr_type=None, attr=None, required: bool = True, exclude: Iterable = [], **kwargs):
```

Comment:
Can you do: `exclude: Iterable = NOT_SET`.

See https://stackoverflow.com/q/10003454

Reply:
✅